modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Features/IdentityFeatureCheckerExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Volo.Abp.Features;
+
+namespace Volo.Abp.Identity.Features
+{
+    public static class IdentityFeatureCheckerExtensions
```

Comment:
An internal extension method would be better to not add a global extension to a framework interface that is only used by the identity module. You can use `InternalsVisibleTo` attribute to share this internal to other identity assemblies.